candid/evm_rpc.did

```diff
@@ -76,6 +76,30 @@ type LogEntry = record {
   removed : bool;
 };
 type Message = variant { Data : vec nat8; Hash : vec nat8 };
+type MultiRpcResult = variant {
+  Consistent : Result;
+  Inconsistent : vec record { RpcService; Result };
+};
+type MultiRpcResult_1 = variant {
+  Consistent : Result_1;
+  Inconsistent : vec record { RpcService; Result_1 };
```

Comment:
Makes sense for sure. This Candid file is automatically generated, but we could manually overwrite these names. The missing piece is a way to statically verify that the Candid interface matches the Rust source code. @dfx-json mentioned that a solution is currently in progress for this (as of a month or two ago).

Reply:
The way it was done for ckETH and some other canisters is to have the 2 (candid and Rust code) not auto-generated but then there is [a unit test](https://sourcegraph.com/github.com/dfinity/ic/-/blob/rs/ethereum/cketh/minter/src/main.rs?L648) that controls that both are equivalent. Ideally, the candid file should be the source of truth and it should be able to generate the boilerplate Rust code, but I think that's out-of-scope for now. Maybe one easy way to modify this would be to avoid Rust generics in the types involved in the Candid interface and use struct like `MultiFeeHistoryResult`/`FeeHistoryResult` so that the generated Candid file is fixed.

Reply:
Thanks! This unit test is exactly what I was looking for. Included in #105.